packages/react-scripts/config/webpack.config.js

```diff
@@ -31,6 +31,7 @@ const getClientEnvironment = require('./env');
 const ModuleNotFoundPlugin = require('react-dev-utils/ModuleNotFoundPlugin');
 const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpackPlugin');
 const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
+const HtmlWebpackEsmodulesPlugin = require('react-dev-utils/html-webpack-esmodules-plugin');
```

Comment:
This path looks incorrect. We should also rename the plugin to be pascal case like the other plugins such as `InlineChunkHtmlPlugin`.

Reply:
Maybe `HtmlWebpackEsModulesPlugin.js`?

Reply:
Will rename and check path